auto_route/pubspec.yaml

```diff
@@ -3,6 +3,8 @@ description: AutoRoute is a declarative routing solution, where everything neede
 version: 3.1.3
 homepage: https://github.com/Milad-Akarie/auto_route_library
 
+publish_to: none
```

Comment:
Same as above. Just to keep track of it. 🙂

Reply:
Ditto